contracts/strategy/NonUSTStrategy.sol

```diff
@@ -4,49 +4,53 @@ pragma solidity =0.8.10;
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
-import "./curve/ICurve.sol";
-import "./BaseStrategy.sol";
-import "./IERC20Detailed.sol";
+
+import {ICurve} from "./curve/ICurve.sol";
+import {IERC20Detailed} from "./IERC20Detailed.sol";
+import {BaseStrategy} from "./BaseStrategy.sol";
 
 /**
- * Strategy that handles non-UST tokens, by first converting them to UST via
+ * EthAnchor Strategy that handles non-UST tokens, by first converting them to UST via
  * Curve (https://curve.fi/), and only then depositing into EthAnchor
  */
 contract NonUSTStrategy is BaseStrategy {
     using SafeERC20 for IERC20;
 
     event Initialized();
 
-    // address of the Curve pool to use
-    ICurve public immutable curvePool;
+    // UST / USDC / USDT / DAI curve pool address
+    ICurve public curvePool;
 
-    // index of the underlying token in the pool
-    int128 public immutable underlyingI;
+    // index of the underlying token in the curve pool
+    int128 public underlyingI;
 
-    // index of the UST token in the pool
-    int128 public immutable ustI;
```

Comment:
why are these no longer immutable? I don't see them being changed anywhere else

Reply:
it's because of testnet strategy contracts.
https://github.com/sandclock-org/solidity_contracts/blob/update/strategy/contracts/test/strategy/TestNonUSTAnchorStrategy.sol#L15

Testnet contract require inherit `NonUSTStrategy`, to make PR simple, there is no testnet contracts yet,
but when we add them, we need to remove immutable.